ironmon_tracker/data/MoveData.lua

```diff
@@ -1,4 +1,6 @@
-MoveData = {}
+MoveData = {
+	totalMoves = 354,
```

Comment:
any particular reason to move the length to a hardcoded variable instead of using # to get the length?

I don't forsee us adding/removing moves/pokemon to be fair so it's not an issue, but if we were to then would also need to remember to update the totalMoves/totalPokemon variables. I'm just curious as to why you felt this change was needed

Reply:
I want to say I opted to do this because using `#` to get the size of the table might perform a calculation that has to determine how big the table is, then return that value; as opposed to us just setting the value here and using it. Maybe with Lua that is not the case.